CHANGES.md

```diff
@@ -1,3 +1,45 @@
+## edge-20.2.2
+
+This release includes the results from continued profiling & performance
+analysis on the Linkerd proxy. In addition to modifying internals to prevent
+unwarranted memory growth, new metrics were introduced to aid in debugging and
+diagnostics. Also, Linkerd's CNI plugin is out of experimental!
```

Comment:
TIOLI: Move the CNI plugin announcement to the 2nd paragraph, and add the sentence: _Check out the docs at https://linkerd.io/2/features/cni/!_